pkg/agent/common/svid/svid.go

```diff
@@ -0,0 +1,38 @@
+package svid
```

Comment:
Every import aliases this to svid_util. I don't really favor "*util", but shouldn't this be named as it will be imported and called most commonly?